src/python/zquantum/qaoa/problems/vertex_cover.py

```diff
@@ -5,42 +5,48 @@
 
 
 class VertexCover(Problem):
-    def __init__(self, hamiltonian_factor: int = 5):
-        self._hamiltonian_factor = hamiltonian_factor
+    """Solves vertex cover problem on an undirected graph using an ising model
+    formulation.
+
+    The solution of a vertex cover problem is the minimal number of colored
+    verticies such that all edges connect to a colored vertex.
+    From "Ising formulations of many NP Problems" by A. Lucas, eq. 33 and 34
+    (https://arxiv.org/pdf/1302.5843.pdf)
+    and
+    https://quantumcomputing.stackexchange.com/questions/16082/vertex-cover-mappings-from-qubo-to-ising-and-vice-versa
+    for corrective translation shifts
+
```

Comment:
Add info about args here as well.
Put explanation of `_A` and `_B` to args, and refer to that in docs for `_A` and `_B`.

Reply:
Made the suggested changes.